src/menu/compassmenustate.cpp

```diff
@@ -126,7 +126,7 @@ CompassMenuState::CompassMenuState(const std::shared_ptr<MenuRingTransform>& rin
   auto addAmmoStats = [this, &inv](const char* weaponName, engine::WeaponType weaponType, size_t& y0)
   {
     const auto& ammo = inv.getAmmo(weaponType);
-    if(inv.count(ammo.weaponType) == 0 && ammo.ammo && ammo.hits == 0 && ammo.hitsTotal == 0 && ammo.misses == 0
+    if(inv.count(ammo.weaponType) == 0 && ammo.shots && ammo.hits == 0 && ammo.hitsTotal == 0 && ammo.misses == 0
```

Comment:
```suggestion
    if(inv.count(ammo.weaponType) == 0 && ammo.shots != 0 && ammo.hits == 0 && ammo.hitsTotal == 0 && ammo.misses == 0
```

Forgot to write an explicit comparison there when I wrote this.